src/utils/chainIdToNetworkName.ts

```diff
@@ -0,0 +1,20 @@
+import { ChainID } from "../enums"
+
+type NetworkMap = {
+  [chainId: number]: string
+}
+
+const networkMap: NetworkMap = {
+  [ChainID.Ethereum]: "Ethereum",
+  [ChainID.Ropsten]: "Ropsten Test",
+  // 4: "Rinkeby",
+  // 5: "Goerli",
+  // 42: "Kovan",
+}
```

Comment:
Based on comment above:

```suggestion
const networkMap: NetworkMap = {
  [ChainID.Ethereum]: "Ethereum",
  [ChainID.Ropsten]: "Ropsten Test",
  // 4: "Rinkeby",
  // 5: "Goerli",
  // 42: "Kovan",
  [ChainId.Localhost]: "Localhost",
}
```